src/main/resources/interface-specification.yml

```
@@ -3929,6 +3963,23 @@ components:
           type: boolean
         activeDescriptor:
           $ref: '#/components/schemas/CompactDescriptor'
+    CompactAgreements:
+      type: object
+      required:
+        - id
+        - eservice
+        - consumer
+        - upgradable
```

As per specifications
```suggestion
        - canBeUpgraded
```